pyi.py

```diff
@@ -1137,15 +1137,15 @@ def visit_Call(self, node: ast.Call) -> None:
         ):
             return self.error(node, Y056.format(method=f".{function.attr}()"))
 
-        # String literals can appear in positional arguments for
-        # TypeVar definitions.
-        with self.string_literals_allowed.enabled():
-            for arg in node.args:
-                self.visit(arg)
+        # String literals can appear as the first positional argument for
+        # TypeVar/ParamSpec/TypeVarTuple/NamedTuple/TypedDict/NewType definitions, etc.
+        if node.args:
+            with self.string_literals_allowed.enabled():
+                self.visit(node.args[0])
         # But in keyword arguments they're most likely TypeVar bounds,
```

Comment:
Nit: This applies to remaining positional arguments too, not just keyword arguments.

```suggestion
        # But in other arguments they're most likely TypeVar bounds,
```

Reply:
good catch!